.eslintrc.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     "flowtype"
   ],
   "env": {
+    "browser": true,
```

Comment:
you are changing the eslint settings for the whole project, which should be runnable on the server (universal React) to allow browser variables

Reply:
Hmm.. I'm not sure how we can have environment-dependent ESLint rules, unless there is some kind of environment variable we can check. We _can_ enable the browser environment inline using:

`/* eslint-env browser */`

But I think our best bet here is probably to just disable ESLint on the relevant lines using:

`// eslint-disable-next-line no-undef`

Let me know your thoughts.

Reply:
There are a few ways to add eslint rules that only apply to certain file names or folders